lib/awful/widget/tasklist.lua.in

```diff
@@ -27,13 +27,13 @@ tasklist.filter = {}
 local function tasklist_label(c, args)
     if not args then args = {} end
     local theme = beautiful.get()
-    local fg_normal = args.fg_normal or theme.tasklist_fg_normal or theme.fg_normal or "#ffffff"
+    local fg_normal = util.check_pango_color(args.fg_normal or theme.tasklist_fg_normal or theme.fg_normal) or "white"
     local bg_normal = args.bg_normal or theme.tasklist_bg_normal or theme.bg_normal or "#000000"
-    local fg_focus = args.fg_focus or theme.tasklist_fg_focus or theme.fg_focus
+    local fg_focus = util.check_pango_color(args.fg_focus or theme.tasklist_fg_focus or theme.fg_focus)
```

Comment:
Those all fall back to `fg_normal` in the if-elses later. That is the current behavior as well.